src/components/HarvestersDialog/index.css

```
@@ -0,0 +1,3 @@
+#harvesters-dialog .Dialog__content {
+    width: 700px;
+  }
```

Indentation is a little off. Don’t you have Prettier installed?
```suggestion
#harvesters-dialog .Dialog__content {
    width: 700px;
  }
```